srcs/app/config/analyser/ParsingState.cpp

```diff
@@ -0,0 +1,47 @@
+#include <parser/ParsingState.hpp>
```

Comment:
I like that you added the parsing state here. It makes a lot more sense now. Thank you!

Reply:
Thx!